### PR TITLE
Approved relationships

### PR DESCRIPTION
**Pull request by goose-life**

### CODE REVIEW

File: indigo_api/models/works.py [hunk 1]
```diff
@@ -34,12 +34,14 @@ class WorkManager(models.Manager):
 
     def get_queryset(self):
         # defer expensive or unnecessary fields
-        return super(WorkManager, self) \
-            .get_queryset() \
+        return super().get_queryset() \
             .select_related('updated_by_user', 'created_by_user', 'country',
                             'country__country', 'locality', 'publication_document') \
             .prefetch_related('commencements')
```
Comment (longhotsummer) [inline indigo_api/models/works.py:40]: commencements are potentially huge, I'm not sure we ALWAYS want to get them whenever we load a work. Can this prefetch be moved to where it's needed?

File: indigo_api/models/works.py [hunk 2]
```diff
@@ -34,12 +34,14 @@ class WorkManager(models.Manager):
...
 
+    def approved(self):
```
Comment (longhotsummer) [inline indigo_api/models/works.py:42]: this should be WorkQuerySet above, so you don't need to get another queryset.

File: indigo_api/models/works.py (hunk 1 above)
Comment (goose-life) [reply indigo_api/models/works.py:40]: I've opened https://github.com/laws-africa/indigo/pull/2114 for this

Comment (longhotsummer) [reply indigo_api/models/works.py:40]: sorry, I don't know how I misread this as a new addition 🤦

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**